### PR TITLE
connect book profile and book list

### PR DESCRIPTION
**Pull request by ZPoreba**
description field (mockDescription variable) is mock temporarily until field won't be added on backend

### CODE REVIEW

File: client/src/BookProfile/BookProfile.js [hunk 1]
```diff
@@ -1,14 +1,23 @@
...
+import { Card, Comment, List, Layout, Form, Button, Input, Spin } from 'antd';
 import './BookProfile.css';
 import Rater from 'react-rater';
 import { withRouter } from "react-router-dom";
 import defaultBook from '../pictures/bookSmile.png';
 import moment from 'moment';
-
+import { bookProfileService } from './services/BookProfileService';
+
+// TODO - delete after field implemented on backend
```
[the remark this answers is not in the document]
Comment (ZPoreba) [reply client/src/BookProfile/BookProfile.js:12]: It's todo comment - will be removed after changing mock to real data

[the remark this answers is not in the document]
Comment (ZPoreba) [reply client/src/BookProfile/BookProfile.js:12]: "description field (mockDescription variable) is mock temporarily until field won't be added on backend" in description

[the remark this answers is not in the document]
Comment (SzymonSmykala) [reply client/src/BookProfile/BookProfile.js:12]: as you wish :P

**Comment by SzymonSmykala** [comment]
@wiktorks @PaulGrishunin please review

**Pull request merged**